mongodb_store/scripts/mongodb_play.py

```diff
@@ -156,9 +156,17 @@ def run(self, running):
         self.event.wait()
 
         timeout = 1
+        try:
+            action_goal_match = True if "Goal" in self.msg_type else False
```

Comment:
do a exact match rather than "Goal" to avoid other message types had Goal in their names